packages/component-library/icons/minimize.icon.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="20" height="20" viewBox="0 0 20 20" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path d="M19 2.269L14.239 7.03L17.2 10H10V2.8L12.961 5.761L17.731 1L19 2.269ZM2.269 19L7.03 14.239L10 17.2V10H2.8L5.761 12.961L1 17.731L2.269 19Z" fill="black"/>
```

Comment:
```suggestion
<path d="M19 2.269L14.239 7.03L17.2 10H10V2.8L12.961 5.761L17.731 1L19 2.269ZM2.269 19L7.03 14.239L10 17.2V10H2.8L5.761 12.961L1 17.731L2.269 19Z" fill="#000000"/>
```

Reply:
@martindavid You can edit it manually, no problem. The problem is that this shouldn't even be an issue for you – I'm looking for the code we use to do the transform. It should accept `black` if it doesn't already.

Figma should ideally also have `#000000` instead of `black` for the icon so that when it gets exported next time it doesn't run into the same issue. Usually it's a hex value though.